src/ol/geom.js

```diff
@@ -4,6 +4,7 @@
 
 export {default as Circle} from './geom/Circle.js';
 export {default as Geometry} from './geom/Geometry.js';
+export {default as GeometryCollection} from './geom/GeometryCollection';
```

Comment:
```suggestion
export {default as GeometryCollection} from './geom/GeometryCollection.js';
```
Adding `.js` like the others exports